src/FSharp.Compiler.Interactive.Settings/FSharp.Compiler.Interactive.Settings.fsproj

```diff
@@ -9,6 +9,11 @@
     <AllowCrossTargeting>true</AllowCrossTargeting>
   </PropertyGroup>
 
+  <!--  The FSharp.Compiler.Interactive.Settings dll provides a referencable public interface to tool builders  -->
+  <PropertyGroup Condition="'$(Configuration)' != 'Proto'">
+      <CompressMetadata>true</CompressMetadata>
```

Comment:
This doesn't look correct. This should have a condition `$(CompressAllMetadata)' == 'true'